### DISCUSSION TITLE
`dynamicFilteringEnabled: true` without `advancedUserEnabled: true` in reported config

### DISCUSSION THREAD

**Discussion by u-RraaLL** [uBlockOrigin/uBlock-issues#1872] [Q&A] [answered]
I'm just curious how this can happen: https://github.com/uBlockOrigin/uAssets/issues/10825 https://github.com/uBlockOrigin/uAssets/issues/10863

My config only shows `advancedUserEnabled: true` without `dynamicFilteringEnabled: true` and I saw other configs reporting the same.
But I've also seen a number of configs showing both settings are `true`.

Is there a reason for these 3 different states of reporting dynamic filtering in the config?

**Comment by ghost** [comment]
This variable no longer exists in the source beside being set to false as default in one place.

**Reply by ghost** [reply]
It had the state of the overview panel in old UI. Now it's  saved `popupPanelSections`.

**Reply by u-RraaLL** [reply]
Yes, I found it earlier in [background.js#L101](https://github.com/gorhill/uBlock/blob/f9e4a109333baf8e2db2719fcda6a3ea5c5767a3/src/js/background.js#L101), but that doesn't exactly answer my question about 3 different configs that can be found in recent uAsset issues:

1. `advancedUserEnabled: true`,  `dynamicFilteringEnabled: true`.
2. `advancedUserEnabled: true`.
3. `dynamicFilteringEnabled: true`.

**Comment by uBlock-user** [comment, marked as answer]
https://github.com/gorhill/uBlock/commit/7a908d293f7699aa27f438a60bb3690cf6f48e10
